resources/careconnect-to-ukcore/allergyintolerance/expected/AllergyIntoleranceDescriptionId-Extension-3to4_001.json

```diff
@@ -0,0 +1,33 @@
+{
+  "resourceType": "AllergyIntolerance",
+  "code": {
+    "coding": [
+      {
+        "extension": [
+          {
+            "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-CodingSCTDescId",
+            "extension": [
+              {
+                "url": "descriptionId",
+                "valueId": "2579700012"
```

Comment:
This should be a valueIdentifier, i.e

             "valueIdentifier": {
               "system": "http://snomed.info/sct",
               "value": "2579700012"
             }